packages/did-ui-react/src/components/CodeVerify/index.component.tsx

```diff
@@ -59,7 +59,7 @@ export default function CodeVerify({
     async (code: string) => {
       try {
         if (code && code.length === 6) {
-          if (!_verifierSessionId) throw Error('Missing verifierSessionId!!!');
+          if (!_verifierSessionId) throw Error(`VerifierSessionId(${_verifierSessionId}) is invalid`);
           setLoading(true);
           const result = await verification.checkVerificationCode({
             verifierSessionId: _verifierSessionId,
```

Comment:
line 116 resendCode
Please return finally message or console.warning for developers.